views/index.ejs

```diff
@@ -92,11 +92,30 @@
     <section class="slice clearfix">
         <div class=" clearfix">
             <div class="container" id="threeColumns">
-                <% if (openhabMajorVersion == '2') { %>
-                    You are using openHAB 2.x. <a href="https://home.myopenhab.org/start/index">Click here to access your openHAB's dashboard</a>
-                <% } else { %>
-                    You are using openHAB 1.x. To remotely access your openHAB's web interface go to <code>https://myopenhab.org/openhab.app?sitemap=yoursitemapname</code>
-                <% } %>
+                <%
+                    switch (openhabstatus) {
```

Comment:
I think this is set to undefined if an openHAB has never connected, not that it has not been setup in the account options.  The registration process forces the use to enter something for the UUID and secret.  So it could be that the OH  binding is not yet running, or that there is a mismatch between the UUID/secret that the user has entered and what's on their system.